integration/lib/php/Boot.class.php

```diff
@@ -218,7 +218,7 @@ function _hx_char_at($o, $i) { $c = substr($o, $i, 1); return FALSE === $c ? ''
 
 function _hx_char_code_at($s, $pos) {
 	if($pos < 0 || $pos >= strlen($s)) return null;
-	return ord($s{$pos});
+	return ord($s[$pos]);
```

Comment:
As we discussed elsewhere, this is Haxe-generated code so these changes won't last. Nonetheless we can merge these to make a release, as a temporary measure, and later on look at how to fix this from the Haxe sources themselves. We should take note of this fact to keep it in mind.

Reply:
Yes, you're right.
We'll need to identify where this code comes from and fix it to avoid it'll come back.
Good point, Xavier. Thanks!